<launch>

  <!-- Launch rviz.launch to visualize this demo -->

  <!-- Debug -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <!-- Verbose mode -->
  <arg name="verbose" default="false" />
  <arg unless="$(arg verbose)" name="command_args" value="" />
  <arg     if="$(arg verbose)" name="command_args" value="--verbose" />

  <!-- PANDA -->
  <include file="$(find moveit_grasps)/launch/load_panda.launch">
  </include>

  <!-- Start the test -->
  <node name="moveit_grasps_demo" launch-prefix="$(arg launch_prefix)" pkg="moveit_grasps"
  type="moveit_grasps_grasp_generator_demo" output="screen" args="$(arg command_args)">
    <param name="ee_group_name" value="hand"/>
    <param name="planning_group_name" value="panda_arm"/>
    <rosparam command="load" file="$(find moveit_grasps)/config_robot/panda_grasp_data.yaml"/>
    <rosparam command="load" file="$(find moveit_grasps)/config/moveit_grasps_config.yaml"/>
  </node>

</launch>
